examples/batch_request.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+# Copyright (C) 2015 Twitter, Inc.
+
+require 'twitter-ads'
+include TwitterAds::Enum
+
+CONSUMER_KEY        = 'your consumer key'.freeze
+CONSUMER_SECRET     = 'your consumer secret'.freeze
+ACCESS_TOKEN        = 'user access token'.freeze
+ACCESS_TOKEN_SECRET = 'user access token secret'.freeze
+ADS_ACCOUNT         = 'ads account id'.freeze
+
+# initialize the client
+client = TwitterAds::Client.new(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
+
+# load the advertiser account instance
+account = client.accounts(ACCOUNT_ID)
```

Comment:
`examples/batch_request.rb:17:in `<main>': uninitialized constant ACCOUNT_ID (NameError)`